### PR TITLE
Add Allegro

### PR DESCRIPTION
**Pull request by vonProteus**

### CODE REVIEW

**Review by NullPiotrException — COMMENTED** [review]
I can confirm that Allegro added SMS 2FA and then software 2FA. Used both of them and they work. Allegro is a Polish Ebay so it fits Retail category.

**Review by NullPiotrException — APPROVED** [review]
I can confirm that Allegro added SMS 2FA and then software 2FA. Used both of them and they work. Allegro is a Polish Ebay so it fits Retail category.

File: _data/retail.yml
Comment (einsteinsfool) [inline _data/retail.yml]: Actually, why did you remove the lang attribute? Does Allegro support English? If so, could you provide a link?
I could only find English version of the API but it's not enough.

**Review by NullPiotrException — DISMISSED** [review]
Ah, forgot that "'lang' should NOT be present when tfa: true.".
Sorry... making a mess out of this PR.

**Review by pawelci — DISMISSED** [review]

**Comment by Carlgo11** [comment]
@pawelci please don't add reviews to PRs. It just creates unnecessary notifications and doesn't actually do anything.

**Comment by NullPiotrException** [comment]
@Carlgo11 I thought that the reason this PR hasn't been merged yet is that some proof is needed. And that can be provided by Allegro user, which I am (I also made a review). But if so, how does the process of reviewing a PR work? Will somebody with merge rights make account in Allegro? Because this PR hasn't been merged for 3 months and newer PRs have been.

**Comment by Carlgo11** [comment]
@einsteinsfool the PR has to be approved and merged by a maintainer of this project. Non-maintainer approvals don't do anything.

**Comment by NullPiotrException** [comment]
@Carlgo11 I know they don't do anything from git's level. But from maintainer perspective, he can either make account on a website himself (which for Allegro requires giving your address, receiving a paper letter to the address and logging from these credentials; or at least it used to a few years ago) or he can trust somebody. And with option 2, it's better to trust multiple people. So the more people approve a request, the more probable it is that it's legitimate.
Since this PR is not merged, my conclusion was that the contributor is not trusted so I made a review.
To make it even more probable, I'll add a screenshot of the login page after pasting the password (red text is my translation).
allegro

**Comment by Carlgo11** [comment]
@einsteinsfool from a maintainer perspective (I'm the chief maintainer) it sends unnecessary notifications.
The reason this PR hasn't been merged yet is because we haven't had time to look at it.

**Comment by NullPiotrException** [comment]
@Carlgo11 I see. Thanks for informing. Won't make more reviews in this repo not to waste your time then.

**Comment by Carlgo11** [comment]
@einsteinsfool thank you!

**Comment by NullPiotrException** [comment]
Shouldn't you merge PRs from the oldest? This one is months old and some new ones have already been merged. It's in Alexa's top300 global ranking.

**Comment by Carlgo11** [comment]
We don’t merge PRs in any particular order.
If something is easy to merge, aka easily verifiable, then we sometimes choose to merge that before we merge another PR that’s harder to merge.

We’ll get to your PR, just be patient.

**Review by Carlgo11 — APPROVED** [review]

**Review by dsoegijono — APPROVED** [review]
Trusting @Carlgo11 on this 😉

**Pull request merged**
